sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/ExpressionInfo.java

```diff
@@ -192,7 +192,7 @@ public ExpressionInfo(String className, String db, String name) {
      * @deprecated This constructor is deprecated as of Spark 3.0. Use other constructors to fully
      *   specify each argument for extended usage.
      */
-    @Deprecated
+    @Deprecated(since = "3.0.0")
```

Comment:
Is there any that can be marked as `forRemoval = true`? For example, `since = "3.0.0"`?